Source/Editor/Editor.cs

```diff
@@ -297,8 +298,14 @@ internal void Init(bool isHeadless, bool skipCompile)
             _modules.Sort((a, b) => a.InitOrder - b.InitOrder);
             _isAfterInit = true;
 
+            // Options module has to init first for localization.
+            _modules[0].OnInit();
+
+            Log("Launching localization service...");
+            EditorLocalization.Manager.SelectLanguage(Options.Options.Interface.Language);
```

Comment:
Maybe initialize `EditorLocalization` after loading the options in `OptionsModule`?